README.md

```diff
@@ -82,21 +82,21 @@ Example using the default output path for the jar:
 #### Options File
 The options file is described in json (example in `specs/options.json`), and has the following possible values:
 
-|Option|Type|Possible Values|
-|---|---|---|
-|validateInfoLicense|boolean|`true`, `false`|
-|validateInfoDescription|boolean|`true`, `false`|
-|validateInfoContact|boolean|`true`, `false`|
-|validateOperationOperationId|boolean|`true`, `false`|
-|validateOperationDescription|boolean|`true`, `false`|
-|validateOperationTag|boolean|`true`, `false`|
-|validateOperationSummary|boolean|`true`, `false`|
-|validateModelPropertiesExample|boolean|`true`, `false`|
-|validateModelNoLocalDef|boolean|`true`, `false`|
-|validateNaming|boolean|`true`, `false`|
-|pathNamingStrategy|string|`CamelCase`, `HyphenCase`, `UnderscoreCase`|
-|parameterNamingStrategy|string|`CamelCase`, `HyphenCase`, `UnderscoreCase`|
-|propertiesNamingStrategy|string|`CamelCase`, `HyphenCase`, `UnderscoreCase`|
+|Option|Type|Possible Values|Description|
+|---|---|---|---|
+|validateInfoLicense|boolean|`true`, `false`|ensures that there is a license section in the info section|
```

Comment:
Can you put a capital letter for each description?

Reply:
Done